src/twisted/web/_element.py

```diff
@@ -31,7 +41,7 @@ class Expose:
     on the class object of which they are methods.
     """
 
-    def __call__(self, *funcObjs: Callable) -> Callable:
+    def __call__(self, f: _Tc, /, *funcObjs: Callable[..., object]) -> _Tc:
```

Comment:
if we don't like the new error message for the TypeError - we can use an `@override` instead